freqtrade/freqtradebot.py

```diff
@@ -237,16 +237,25 @@ def _refresh_whitelist(self, whitelist: List[str]) -> List[str]:
 
         return final_list
 
-    def get_target_bid(self, ticker: Dict[str, float]) -> float:
+    def get_target_bid(self, pair: str) -> float:
         """
         Calculates bid target between current ask price and last price
         :param ticker: Ticker to use for getting Ask and Last Price
         :return: float: Price
         """
-        if ticker['ask'] < ticker['last']:
-            return ticker['ask']
-        balance = self.config['bid_strategy']['ask_last_balance']
-        return ticker['ask'] + balance * (ticker['last'] - ticker['ask'])
+
+        if self.config['bid_strategy']['use_book_order']:
+            logger.info('Using order book ')
+            orderBook = exchange.get_order_book(pair)
+            return orderBook['bids'][self.config['bid_strategy']['use_book_order']][0]
```

Comment:
did u mean `book_order_top` instead of `use_book_order `? Otherwise it looks like:
`orderBook['bids'][true][0]`

Reply:
I agree with Arudov, it looks like a typo

Reply:
Thanks! this is a typo. This has been fixed with my local version.

Reply:
would it make sense to make the `[0]` configurable?

it may be benefitial to not use the first bid-price, but 1-2 lower (assuming a quick market ... ?)
at least that's what i heared from the slack discussions ...

Reply:
@xmatthias [0] is a price and [1] is a size. The parameter you are taking about is `use_book_order`, which should be `book_order_top`